### PR TITLE
add clientName and heartBeat metadata for pubsub rabbitmq

### LINKED ISSUE #3710: Add docs for RabbitMQ pub/sub `connection_name`
This issue was automatically created by [Dapr Bot](https://github.com/dapr/dapr/blob/master/.github/workflows/dapr-bot.yml) because a "documentation required" label was added to dapr/components-contrib#2902. 

TODO: Add more details as per [this template](.github/ISSUE_TEMPLATE/new-content-needed.md).

### PR DESCRIPTION
**Pull request by Taction**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add missing content.

## Issue reference

Please reference the issue this PR will close: #3710

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md [hunk 1]
```diff
@@ -96,6 +100,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | caCert | Required for using TLS | Input/Output | Certificate Authority (CA) certificate in PEM format for verifying server TLS certificates. | `"-----BEGIN CERTIFICATE-----\n<base64-encoded DER>\n-----END CERTIFICATE-----"`
 | clientCert  | Required for using TLS | Input/Output | TLS client certificate in PEM format. Must be used with `clientKey`. | `"-----BEGIN CERTIFICATE-----\n<base64-encoded DER>\n-----END CERTIFICATE-----"`
 | clientKey | Required for using TLS | Input/Output | TLS client key in PEM format. Must be used with `clientCert`. Can be `secretKeyRef` to use a secret reference. | `"-----BEGIN RSA PRIVATE KEY-----\n<base64-encoded PKCS8>\n-----END RSA PRIVATE KEY-----"`
+| clientName | N | RabbitMQ [client-provided connection name](https://www.rabbitmq.com/connections.html#client-provided-names), this a custom identifier. If set, the identifier will be mentioned in RabbitMQ server log entries and management UI. Can be set to {uuid}, {podName} or {appID} which will be replaced by Dapr runtime to the real value. | `"app1"`, `{uuid}`, `{podName}`, `{appID}`
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md]:
```suggestion
| clientName | N | This RabbitMQ [client-provided connection name](https://www.rabbitmq.com/connections.html#client-provided-names) is a custom identifier. If set, the identifier will be mentioned in RabbitMQ server log entries and management UI. Can be set to {uuid}, {podName}, or {appID}, which is replaced by Dapr runtime to the real value. | `"app1"`, `{uuid}`, `{podName}`, `{appID}`
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
quick review - thanks for this PR!

**Comment by msfussell** [comment]
@Taction please look at the comment and resolve conflicts

**Comment by hhunter-ms** [comment]
resolved conflicts

**Comment by Taction** [comment]
@hhunter-ms Thanks for your review and for resolving conflicts!

**Comment by msfussell** [comment]
@Taction - This PR is in the v1.11 docs, but the code is in v1.12. Can you move this PR to target the v1.12 docs? Thanks

**Comment by Taction** [comment]
@msfussell Thanks for pointing out that. I opened another pr #3740 to target the v1.12 doc.

**Pull request closed without merging**